src/components/Layout/Header/Header.module.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../variables.scss';
+
+*{
+    padding: 0;
+    margin: 0;
+}
+
+.c-header {
+    background-color: $color-blue;
+    color: $color-white;
+    width: 100%;
+    padding: $space-l $space-l;
+    text-align: center;
+    vertical-align: middle;
+    &__logo{
```

Comment:
Good job with naming classes, one thing to look out is spacing add space before `&` section and before opening bracket. Do this for all classes below.